src/igmpproxy.h

```diff
@@ -113,7 +113,7 @@ void my_log( int Serverity, int Errno, const char *FmtSt, ... );
 
 /* ifvc.c
  */
-#define MAX_IF         40     // max. number of interfaces recognized
+#define MAX_IF         100     // max. number of interfaces recognized
```

Comment:
I will remove this entire commit from the pull request. 

This change is also done in pfSense because 40 is too low. But agree with you that this should be fixed the right way for a seperate pull requests.